db/schema.rb

```diff
@@ -11,21 +11,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200207120244) do
+ActiveRecord::Schema.define(version: 20200207141946) do
```

Comment:
There are a lot of changes here. I wonder if you should do clean db migration and see if you get a different result. Looks like stuff from another branch.

Reply:
Yes, I will run migration with clean db and update it. Thanks. :)